qdrant-landing/content/documentation/integrations/mindsdb.md

```diff
@@ -0,0 +1,87 @@
+---
+title: MindsDB
+weight: 1100
+---
+
+# MindsDB
+
+[MindsDB](https://mindsdb.com) is an AI automation platform for building AI/ML powered features and applications. It works by connecting any source of data with any AI/ML model or framework and automating how real-time data flows between them.
+
+With the MindsDB-Qdrant integration, you can now select Qdrant as a database to load into and retrieve from with semantic-search and filtering.
+
+**MindsDB allows you to easily**:
+
+ - Connect to any store of data or end-user application.
+ - Pass data to an AI model from any store of data or end-user application.
+ - Plug the output of an AI model into any store of data or end-user application.
+ - Fully automate these workflows to build AI-powered features and applications
+
+
+## Usage
+To get started with Qdrant and MindsDB, the following syntax can be used.
+```sql
+CREATE DATABASE qdrant_test
+WITH ENGINE = "qdrant",
+PARAMETERS = {
+    "location": ":memory:",
+    "collection_config": {
+        "size": 386,
+        "distance": "Cosine"
+    }
+}
+```
+The available arguments for instantiating Qdrant can be found [here](https://github.com/mindsdb/mindsdb/blob/23a509cb26bacae9cc22475497b8644e3f3e23c3/mindsdb/integrations/handlers/qdrant_handler/qdrant_handler.py#L408-L468).
```

Comment:
I took a look at the linked page, and there is one issue we need to fix. There are two local modes of running Qdrant:
- in-memory (enabled by setting the `location` to `:memory:`)
- on-disk (enabled by setting `path`)

Thus, "_Persistence path for a local Qdrant instance(:memory:)._" is a wrong description and might be misleading. If someone sets the `location` to `:memory:` and provides a `path`, the path is going to be ignored:
![image](https://github.com/qdrant/landing_page/assets/2649301/f1a69276-8b60-4fa6-bf35-f9dd67c96faf)

Are we able to fix this?

Reply:
Opened up a PR to resolve this.
https://github.com/mindsdb/mindsdb/pull/8000